### DISCUSSION TITLE
Quick Replies

### DISCUSSION THREAD

**Discussion by VeryUsual** [InfiniTimeOrg/InfiniTime#1739] [Ideas]
When you get a text message, have the ability to reply common replies like "OK", "Yes" or "No" on the watch.

**Comment by LinuxinaBit** [comment]
This requires companion app support which, as I've mentioned in one of your other suggestions, is really slow.
I would love this feature though :)

**Reply by VeryUsual** [reply]
In the Gadgetbridge wiki
[Canned Messages](https://codeberg.org/Freeyourgadget/Gadgetbridge/wiki/Calls-and-SMS#configure-canned-replies)
Replies: define answers to pick from „interactive notifications“, so you can e.g. _answer SMS directly from your Gadget_
